en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +66,86 @@ We will only consider merging post-beta breaking API changes if they are necessa
 
 Non-breaking API changes and enhancements may be merged after the initial CMS 5 beta release.
 
-## Post CMS5 RC freeze
+#### Post CMS5 RC freeze
 Once the initial CMS5.0 RC is released, non-breaking API changes will no longer be accepted and will have to be targeted to CMS5.1.
 
 Bug fixes may be accepted if they can be solved without changing APIs.
+
+### Known issues
+
+The Silverstripe CMS 5 beta 1 release contains several know issues. We'll aim to address those prior the Silverstripe CMS 5 Release candidate. These likely only the tip of the iceberg.
+
+#### Bugs
```

Comment:
```suggestion
#### Regressions
```
There are a ton of bugs, which have been around for centuries. We're only interested in listing new regressions.